esphome/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_ESPHOME_IP
+      APP_PORT: 6052
+
+  server:
+    image: esphome/esphome:2022.6.2@sha256:1fac6334b85e76c7fd936036891b7a7ee761ee73fd119662b2cb564c3faf818f
+    volumes:
+      - ${APP_DATA_DIR}/data:/config
+      # Needed for flashing new ESPs
+      - /dev:/dev
```

Comment:
@ShonP40 Just another thought/idea. Could you check if it is still working using `/dev/bus/usb:/dev/bus/usb` to lock this down further?

And can you also check that it is still working when you plug in a USB device **after** the container has started?

Reply:
ESPHome flashes ESPs through a tty connection though
e.g. `/dev/ttyUSB0`